### ISSUE TITLE
Accessibility Feedback for Design intent [Facilities - Expandable alert for Vet Center operating status]

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24097]
# Accessibility Feedback for Design intent [Facilities - Expandable alert for Vet Center operating status]

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Must
- Provide alternative text to the Alert icon (use `sr-only` text or an `aria-label`) so screen readers have an equivalent understanding of the purpose/nature of the message

## Should
- Ensure there are no skipped headings and that the heading order is logical. For example, option A's status alert _looks_ like the same heading level as its parent, when it should be a heading level under.

## Consider
- Using option B instead of option A. Option A limits space on mobile due to the heavy padding from the alert icon. Option A is also styled as a heading while similar information below it is not. 
- Removing the left-side padding of the accordion's content and using bullets to split up information. This will make content easier to read for Veterans with reading-related or cognitive disabilities e.g.:

![image](https://user-images.githubusercontent.com/14154792/116895035-c16c2680-ac00-11eb-970e-d4c9e7920ed7.png)

## Questions
- Will Veterans understand how to use telehealth services? The copy "we recommend using our telehealth services" does not provide direction on where to go for telehealth services.

**Comment by mmiddaugh** [comment]
Spec will be updated and documented for experimental design system in #24195. Seeking guidance on proper semantic markup is included in the acceptance criteria
